tvtk/wrapper_gen.py

```diff
@@ -511,223 +685,178 @@ def _gen_get_set_methods(self, klass, out):
-                                                 vtk_set_meth)
-                        else: # Get has args or Set needs many args.
-                            self._write_tvtk_method(klass, out, vtk_get_meth, g_sig)
-                            self._write_tvtk_method(klass, out, vtk_set_meth, s_sig)
-                        del updateable_traits[name]
-                elif typ is bool:
-                    t_def = 'traits.Bool(%(default)s)'%locals()
+            # ------------------------------------------------------
+            # The default is None, depending on the get/set methods
+            # signature, we will either have the trait as a Property
+            # and write getter/setter or we would just write the
+            # VTK get and set methods
+            # -------------------------------------------------------
+            if default is None or isinstance(default, vtk.vtkObjectBase):
+                # Bunch of hacks to work around issues.
+                #print get_sig, vtk_get_meth, klass.__name__
+                if len(get_sig) == 0:
+                    get_sig = [([None], None)]
+
+                if len(set_sig) == 0:
+                    set_sig = [([None], [None])]
+                    get_sig = [([None], None)]
+
+                elif set_sig[0][1] is None or set_sig[0][1] == '':
+                    set_sig[0] = list(set_sig[0])
+                    set_sig[0][1] = [None]
+
+                if get_sig[0][0][0] == 'string' and get_sig[0][1] is None:
+                    # If the get method really returns a string
+                    # wrap it as such.
+                    t_def = 'traits.Trait(None, None, '\
+                            'traits.String(enter_set=True, auto_set=False))'
                     self._write_trait(out, name, t_def, vtk_set_meth,
                                       mapped=False)
                 else:
-                    print("%s:"%klass.__name__, end=' ')
-                    print("Ignoring method: Get/Set%s"%m)
-                    print("default: %s, range: None"%default)
+                    if (get_sig[0][1] is None) and (len(set_sig[0][1]) == 1):
+                        # Get needs no args and Set needs one arg
+                        self._write_property(out, name, vtk_get_meth,
+                                             vtk_set_meth)
+                    else: # Get has args or Set needs many args.
+                        self._write_tvtk_method(klass, out, vtk_get_meth, get_sig)
+                        self._write_tvtk_method(klass, out, vtk_set_meth, set_sig)
+
+                    # We cannot update the trait
                     del updateable_traits[name]
 
-            else: # Has a specified range of valid values.
-                if klass.__name__ == 'vtkCubeAxesActor2D' and \
-                       name == 'inertia':
-                    # VTK bug.  Inconsistent API!
-                    rng = (float(rng[0]), float(rng[1]))
```

Comment:
I have removed the special handling of 'vtkCubeAxesActor2D.inertia' here because from what I have researched on the VTK API, `inertia` is indeed an `int` and there is no inconsistency.